input/data/projects/nsubstitute.yml

```diff
@@ -0,0 +1,14 @@
+Source: https://github.com/nsubstitute/NSubstitute
+NuGet:
+  - NSubstitute
+  - NSubstitute.Analyzers.CSharp
+  - NSubstitute.Analyzers.VisualBasic
+Language: C#
+Image: http://nsubstitute.github.io/images/nsubstitute-100x100.png
+Title: NSubstitute
+Description: NSubstitute is a friendly substitute for .NET mocking frameworks. It has a simple, succinct syntax to help developers write clearer tests. NSubstitute is designed for Arrange-Act-Assert (AAA) testing and with Test Driven Development (TDD) in mind.
+Website: https://nsubstitute.github.io/
+Docs: https://nsubstitute.github.io/help.html
+Twitter: nsubstitute
+Tags:
+  - Testing
```

Comment:
Chat:L

Reply:
I'm not sure that counts as chat? I'm thinking of moving away from the group in favour of GH issues and StackOverflow anyway, as that seems to be where most people tend to try first.

Reply:
Oh. Sorry. Yeah, sometimes Gitter/GitHub/StackOverflow can be a bit much.